lib/py/src/TRecursive.py

```diff
@@ -0,0 +1,63 @@
+# MODIFIED June 20, 2017, Eric Conner
+#
+#
+# Original source copyright 2014-present Facebook, Inc.
+#
```

Comment:
Yea, I'm not very experienced with the whole issue of licensing.  I mainly put these lines because of the fbthrift license and I used this file more or less verbatim from fbthrift.  (https://github.com/facebook/fbthrift/blob/master/LICENSE#L97) (https://github.com/facebook/fbthrift/blob/master/thrift/lib/py/util/Recursive.py)

>       (b) You must cause any modified files to carry prominent notices
          stating that You changed the files; and

      (c) You must retain, in the Source form of any Derivative Works
          that You distribute, all copyright, patent, trademark, and
          attribution notices from the Source form of the Work,
          excluding those notices that do not pertain to any part of
          the Derivative Works; and

But perhaps it is fine to use verbatim since fbthrift is itself a branch of thrift?  Not really sure what the protocol here is...